### DISCUSSION TITLE
Backup and restore

### DISCUSSION THREAD

**Discussion by Havouza15** [immich-app/immich#6977] [General]
Some days ago I asked a question about how best migrate Immich to another server. I was pointed to the Backup and Restore dokumentation and that was what I was looking fore.
Just one question.
In the documentation it states that this 3 folders need to be migrated, but the profile folder is not there.
UPLOAD_LOCATION/library
UPLOAD_LOCATION/upload
UPLOAD_LOCATION/profile

**Comment by bo0tzz** [comment]
The profile folder is only created if you upload a profile picture.
